po/ru.po

```diff
@@ -973,54 +991,48 @@ msgstr ""
-
-#: src/core/dbus-unit.c:362 src/core/dbus-unit.c:363
-msgid "Authentication is required to restart '$(unit)'."
-msgstr "Чтобы перезапустить «$(unit)», необходимо пройти аутентификацию."
-
-#: src/core/dbus-unit.c:535
-msgid ""
-"Authentication is required to send a UNIX signal to the processes of "
-"'$(unit)'."
-msgstr ""
-"Чтобы отправить сигнал UNIX процессам юнита «$(unit)», необходимо пройти "
-"аутентификацию."
-
-#: src/core/dbus-unit.c:566
-msgid "Authentication is required to reset the \"failed\" state of '$(unit)'."
-msgstr ""
-"Чтобы сбросить состояние «failed» у юнита «$(unit)», необходимо пройти "
-"аутентификацию."
-
-#: src/core/dbus-unit.c:599
-msgid "Authentication is required to set properties on '$(unit)'."
-msgstr ""
-"Чтобы изменить параметры юнита «$(unit)», необходимо пройти аутентификацию."
-
-#: src/core/dbus-unit.c:708
-msgid ""
-"Authentication is required to delete files and directories associated with "
-"'$(unit)'."
-msgstr ""
-"Чтобы удалить файлы и директории, относящиеся к юниту «$(unit)», необходимо "
-"пройти аутентификацию."
-
-#: src/core/dbus-unit.c:757
-msgid ""
-"Authentication is required to freeze or thaw the processes of '$(unit)' unit."
-msgstr ""
-"Чтобы отправить сигнал заморозки или разморозки процессам юнита «$(unit)», "
-"необходимо пройти аутентификацию."
+#~ msgid "Authentication is required to start '$(unit)'."
+#~ msgstr "Чтобы запустить «$(unit)», необходимо пройти аутентификацию."
+
+#~ msgid "Authentication is required to stop '$(unit)'."
+#~ msgstr "Чтобы остановить «$(unit)», необходимо пройти аутентификацию."
+
+#~ msgid "Authentication is required to reload '$(unit)'."
+#~ msgstr ""
+#~ "Чтобы заставить «$(unit)» перечитать конфигурацию, необходимо пройти "
+#~ "аутентификацию."
+
+#~ msgid "Authentication is required to restart '$(unit)'."
+#~ msgstr "Чтобы перезапустить «$(unit)», необходимо пройти аутентификацию."
+
+#~ msgid ""
+#~ "Authentication is required to send a UNIX signal to the processes of "
+#~ "'$(unit)'."
+#~ msgstr ""
+#~ "Чтобы отправить сигнал UNIX процессам юнита «$(unit)», необходимо пройти "
+#~ "аутентификацию."
+
+#~ msgid ""
+#~ "Authentication is required to reset the \"failed\" state of '$(unit)'."
+#~ msgstr ""
+#~ "Чтобы сбросить состояние «failed» у юнита «$(unit)», необходимо пройти "
+#~ "аутентификацию."
+
+#~ msgid "Authentication is required to set properties on '$(unit)'."
+#~ msgstr ""
+#~ "Чтобы изменить параметры юнита «$(unit)», необходимо пройти "
+#~ "аутентификацию."
+
+#~ msgid ""
+#~ "Authentication is required to delete files and directories associated "
+#~ "with '$(unit)'."
+#~ msgstr ""
+#~ "Чтобы удалить файлы и директории, относящиеся к юниту «$(unit)», "
+#~ "необходимо пройти аутентификацию."
+
+#~ msgid ""
+#~ "Authentication is required to freeze or thaw the processes of '$(unit)' "
+#~ "unit."
+#~ msgstr ""
+#~ "Чтобы отправить сигнал заморозки или разморозки процессам юнита "
+#~ "«$(unit)», необходимо пройти аутентификацию."
```

Comment:
Why remove those?

Reply:
Not sure I can follow, but please untouch or update them, instead of commenting out them.

Reply:
fixed